src/Areas/AzureArc/Services/IArcServices.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+
+namespace AzureMcp.Areas.AzureArc.Services;
+
+public interface IArcServices
+{
+    string LoadResourceFiles(string resourceName);
+    Process StartProcess(string scriptPath, ProcessStartInfo processInfo);
```

Comment:
do we need LoadResourceFiles and StartProcess in the IArcServices, seems they are used only with in ArcService class we can have these as private methods